src/cli.ts

```diff
@@ -23,7 +23,7 @@ import {
   ReturnCode,
   test
 } from './main'
-import { fileExists } from './utils/file'
+import { fileExists } from '@sasjs/utils/file'
```

Comment:
we should import utils from `@sasjs/utils`

Reply:
done